src/libstd/Cargo.toml

```diff
@@ -43,9 +43,12 @@ cc = "1.0"
 build_helper = { path = "../build_helper" }
 
 [features]
+default = ["compiler_builtins_c"]
```

Comment:
Why not just "compiler_builtins/c" here?

Reply:
Because then one cannot enable it from "outside", like in Xargo.